tests/system/python/e2e/test_e2e_notification_service_with_plugins.py

```diff
@@ -272,30 +291,40 @@ def enable_notification(self, foglamp_url, cat_name, is_enabled=True):
         assert 200 == r.status
 
     def test_sent_and_receive_notification(self, foglamp_url, start_south, wait_time):
-        data = {"name": "Test4", "description": "Test4_Notification", "rule": NOTIFY_RULE, "channel": NOTIFY_PLUGIN,
-                "enabled": False, "notification_type": "retriggered"}
+        data = {"name": "Test4", "description": "Test4_Notification", "rule": NOTIFY_INBUILT_RULES[0],
+                "channel": NOTIFY_PLUGIN, "enabled": True, "notification_type": "one shot"}
         name = data['name']
         _add_notification_instance(foglamp_url, data)
         self.configure_rule_with_latest_eval_type(foglamp_url, name)
+
         self.enable_notification(foglamp_url, name)
+        """ In syslog:
+            
+        WARNING: configuration_manager: foglamp.common.configuration_manager: Got an issue while decoding config item: 
+        {'type': 'script', 'file': '', 'description': 'Python 3.5 module to load.', 'order': '2', 'displayName':
+         'Python script', 'value': '""', 'default': '""'} | Non-hexadecimal digit found
+        WARNING: Notification plugin 'python35', called without a Python 3.5 script. 
+        Check 'script' item in 'deliveryTest4' configuration. Notification plugin has been disabled.
+
+        """
```

Comment:
`/foglamp/audit?source=NTFSN` assertion is failing because of above. please check